.github/workflows/stale.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Close stale issues and PRs'
+
+permissions:
+  actions: write
+  issues: write
+  pull-requests: write
+  
+on:
+  schedule:
+    - cron: "30 1 * * *" # once a day (1:30 UTC)
+  workflow_dispatch: # allow manual trigger
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
+          days-before-stale: 30
+          days-before-close: 5
```

Comment:
I propose to close after 7 days, which would be a whole week. Otherwise, the time to react would heavily depend on the day of the week the issue was created.